expenses/views.py

```
@@ -115,36 +124,29 @@ def edit_expense(request, pk):
     expense.save()
 
     new_ids = []
-    for idx, expensePartId in enumerate(request.POST.getlist('expensePartId[]')):
-        budget_lines = request.POST.getlist('budgetLine[]')
-        response = requests.get("https://budget.datasektionen.se/api/budget-lines/{}".format(budget_lines[idx]))
+
+    for idx, budgetLineId in enumerate(request.POST.getlist('budgetLine[]')):
+
+        amount = request.POST.getlist('amount[]')[idx]
+
+        if int(amount) < 1:
+            continue
```

Maybe not fail silently here.